woocommerce/woocommerce
# Fix: Product Collection block does not display when editing template/post

### Changes proposed in this Pull Request:





Closes #47868 



### How to test the changes in this Pull Request:



**Note for Developers:** We were able to reproduce it consistently with "prod" build instead of "dev" build, so I would suggest you to test it on "prod" build. 

**Testing in Product Catalog Template:**
1. From admin go to Appearance > Editor
2. From the design editor go to Templates > Product Catalog (edit)
3. Replace the Products (Beta) block with the new Product Collection block
4. Save the template
5. Now go back to the admin and then back to Appearance > Editor > Templates > Product Catalog (edit)
	- Verify that the Product Collection block is showing a list of products in the Editor. 

**Testing in new post**
1. Create a new post
2. Add Product Collection block to it & choose "Product Catalog" template
3. Save the draft
4. Refresh the post and verify that the Product Collection block displays the products.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Fix: Product Collection block does not display properly when editing template/post

#### Comment 

</details>
